server/controllers/admin/userManagementController.ts

```diff
@@ -39,6 +39,9 @@ export default class UserController {
         hrefPrefix,
         sortBy,
         sortDirection,
+        qualifications: qualification,
```

Comment:
I think it might be clearer if we rename `qualifications` to be singular if the template is expecting there to only be one. Same on line 44

Reply:
Renamed to `selectedQualification` as agreed